CHANGELOG.md

```diff
@@ -1,8 +1,26 @@
 # Changelog
 
+## 1.0.0b5 / 2021-01-13
+
+* [Added] Add new live and rehydrated logs breakdowns for Usage API to public documentation. See [#223](https://github.com/DataDog/datadog-api-client-python/pull/223).
+* [Added] Add support for Synthetics variables from test. See [#214](https://github.com/DataDog/datadog-api-client-python/pull/214).
+* [Added] Add filters to rule endpoints in security monitoring API. See [#209](https://github.com/DataDog/datadog-api-client-python/pull/209).
+* [Added] Add Azure app services fields to usage v1 endpoints. See [#208](https://github.com/DataDog/datadog-api-client-python/pull/208).
+* [Added] Add mobile RUM OS types usage fields. See [#207](https://github.com/DataDog/datadog-api-client-python/pull/207).
+* [Added] Add config variables for synthetics API tests. See [#206](https://github.com/DataDog/datadog-api-client-python/pull/206).
+* [Added] Add endpoints for the public API of Logs2Metrics. See [#203](https://github.com/DataDog/datadog-api-client-python/pull/203).
+* [Added] Add endpoints for API Keys v2. See [#200](https://github.com/DataDog/datadog-api-client-python/pull/200).
+* [Added] Add javascript value to synthetics browser variable types. See [#197](https://github.com/DataDog/datadog-api-client-python/pull/197).
+* [Added] Add synthetics assertion operator. See [#186](https://github.com/DataDog/datadog-api-client-python/pull/186).
+* [Fixed] Fix missing field for synthetics variables from test. See [#220](https://github.com/DataDog/datadog-api-client-python/pull/220).
```

Comment:
Should we kill this since:
```
* [Added] Add support for Synthetics variables from test. See [#214](https://github.com/DataDog/datadog-api-client-python/pull/214).
```
is part of this release too?

Reply:
definitely